doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc

```diff
@@ -0,0 +1,38 @@
+[id='installation-options-ref_{context}']
+= Installation environment options for {PRODUCT}
+
+With {PRODUCT}, you can set up a development environment to develop business applications, a runtime environment to run those applications to support decisions, or both. You can also configure a clustered environment for high availability, enhanced collaboration, and load balancing.
+
+* *Development environment*: Typically consists of one {CENTRAL} installation and at least one {KIE_SERVER} installation. {CENTRAL} enables you to create decision processes, rules, and other artifacts, and {KIE_SERVER} enables you to test the artifacts that you created.
+
+* *Runtime environment*: Consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} has an embedded {CONTROLLER}. If you install {CENTRAL}, use the *Menu* -> *Deploy* -> *Execution servers* page to create and maintain containers. If you want to automate {KIE_SERVER} management without {CENTRAL}, you can use the {HEADLESS_CONTROLLER}.
+
+* *Clustered environment*: Consists of a unified group or "cluster" of two or more computers. You can cluster both development and runtime environments. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration. The primary benefit of clustering a runtime environment is high availability and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster to improve the overall performance.
```

Comment:
This seems kind of out of place format wise. Maybe make this just a paragraph about clustering and high availability. As the way its laid out almost makes it seem like a separate environment from development and runtime.